process_manager.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package processmanager;
+
+service ProcessManager {
+    rpc RunProcess(RunProcessRequest) returns (RunProcessResponse);
+    rpc StopProcess(StopProcessRequest) returns (StopProcessResponse);
+    rpc GetProcessStatus(ProcessStatusRequest) returns (ProcessStatusResponse);
+    rpc GetProcessOutput(ProcessOutputRequest) returns (stream ProcessOutputResponse);
+}
+
+message RunProcessRequest {
+    string command = 1;
+    string user = 2;
+}
+
+message RunProcessResponse {
+    string processId = 1;
+}
+
+message StopProcessRequest {
+    string processId = 1;
+    string user = 2;
+}
+
+message StopProcessResponse {
+    string status = 1;
+}
+
+message ProcessStatusRequest {
+    string processId = 1;
+    string user = 2;
+}
+
+message ProcessStatusResponse {
+    string status = 1;
+}
+
+message ProcessOutputRequest {
+    string processId = 1;
+    string user = 2;
+}
+
+message ProcessOutputResponse {
+    string output = 1;
```

Comment:
Protobuf `string`s must be valid UTF-8, and we can't make assumptions about the process output. I would use `bytes` here instead.